### ISSUE TITLE
goauthentik/client-go: "400 Bad Request" error when trying to set user's token key

### ISSUE THREAD

**Issue by AndriyKalashnykov** [goauthentik/authentik#4551]
**Describe the bug**

When calling  Authentik API to set users token key (token does exist) I get  "400 Bad Request"

```golang
apiClient.CoreApi.CoreTokensSetKeyCreate(ctx, tokenIdentifier).TokenSetKeyRequest(api.TokenSetKeyRequest{
		Key: key,
	}).Execute()
```

see compilable [example](https://github.com/AndriyKalashnykov/authentik-k8s/blob/f66430d14ac36892ca1e947daee14119761c3fda/gotest/internal/authentik/api.go#L68)

**Expected behavior**
Method 

**Logs**
```log
POST /api/v3/core/tokens/qleetctl-token/set_key/ HTTP/1.1
Host: 172.18.255.200:443
User-Agent: OpenAPI-Generator/1.0.0/go
Content-Length: 71
Accept: application/json
Authorization: Bearer NoMlxBQuYgfu3j19ygGqhjXenAjrJgOfN5naqmSDBUhdLjYqHKze7yyzY07H
Content-Type: application/json
Accept-Encoding: gzip

{"key":"ZId4CDEtmHbnuxkJH2ehUzHgYeTmOansuCO0JsTTsZnYB1z9N0WoAutpyH4i"}

2023/01/27 16:59:59 
HTTP/1.1 400 Bad Request
Allow: POST, OPTIONS
Date: Fri, 27 Jan 2023 21:59:58 GMT
Referrer-Policy: same-origin
Vary: Accept-Encoding
Vary: Cookie
X-Authentik-Id: 890e95e9de534724b6af7ec49186c25a
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-Powered-By: authentik


2023/01/27 16:59:59 error: 400 Bad Request

```

**Version and Deployment (please complete the following information):**
 - authentik version: 2023.1.2
 - Deployment: kubernetes (using local [kind](https://kind.sigs.k8s.io/) cluster)

**Additional context**
I need to set existing token key to some predefined value

**Comment by AndriyKalashnykov** [comment]
@BeryJu  The issues in not closed with latest goauthentik.io/api/v3 v3.2023012.3 and  2023.1.2 server

**Comment by BeryJu** [comment]
Because this is a server-side issue, and will be fixed in 2023.2.0
